index.html

```diff
@@ -0,0 +1,213 @@
+            ALL OVER THE GLOBE
+          </h1>
+          <p class="intro-description">
+            Our services covers assistance in availing Scholarships and fee
+            discounts, Assistance in VISA processing & Travel arrangements,
+            Assistance in tuition fee payment and refunds, Assistance in course
+            Selection, and assistance in applying to the universities around the
+            world.
+          </p>
+          <p class="event-date">2022.08.6(SAT) ~ 7(SUN)</p>
+          <small>@ iAmWaris Tower, Kabul.</small>
+        </div>
+      </main>
+    </div>
+    <!-- home and header section end -->
+
+    <!-- Our Programs -->
+    <section class="program">
+      <div class="container">
+        <div class="program-title">
+          <h2>Our Services</h2>
+          <span></span>
+        </div>
+
+        <div class="program-container">
+          <div class="activity">
+            <img src="./assets/img/exhibition.png" alt="lecture" />
+            <h3>Scholarships</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+          <div class="activity">
+            <img src="./assets/img/forum.png" alt="Exhibition" />
+            <h3>VISA Processing</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+          <div class="activity">
+            <img src="./assets/img/ignite.png" alt="Forum" />
+            <h3>Jobs</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+          <div class="activity">
+            <img src=".//assets/img/lecture.png" alt="Workshop" />
+            <h3>Workshop</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+          <div class="activity">
+            <img src="./assets/img/workshop.png" alt="networking" />
+            <h3>Interview Prep</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+        </div>
+        <a href="#" class="program-desktop"> SEE ALL THE SERVICES</a>
+        <a href="#" class="program-mobile"> Join Upcoming Workshop 2022</a>
+      </div>
+    </section>
+
+    <!-- Personnel -->
+    <section class="speakers">
+      <div class="speakers-title">
+        <h2>Key Personnel</h2>
+        <span></span>
+      </div>
+      <div class="keynote-speakers"></div>
+      <button type="button" class="btn" id="btn">
+        See More <span> <i class="fa fa-angle-down"></i></span>
+      </button>
+    </section>
```

Comment:
- Please make sure your images are of the same sizes and the texts beside them are well aligned as seen in the original design.
- Original Design
![screenshot (62)](https://user-images.githubusercontent.com/44624138/183272944-c380b78b-a4c3-415d-8169-a2b9311b5f7b.png)
![screenshot (63)](https://user-images.githubusercontent.com/44624138/183272948-07db7d07-67d9-49d2-893f-7ce7c0a452ce.png)
- Your Design
![screenshot (64)](https://user-images.githubusercontent.com/44624138/183273052-fe37856b-513d-4694-89c0-1259ac9248a7.png)
![screenshot (65)](https://user-images.githubusercontent.com/44624138/183273054-52fb3b6c-3432-48bf-9d5e-af9625ee3111.png)

Reply:
addressed issues were solved